projects/plugins/jetpack/extensions/blocks/ai-assistant/editor.js

```diff
@@ -1,4 +1,39 @@
-import registerJetpackBlock from '../../shared/register-jetpack-block';
-import { name, settings } from '.';
+import { getRedirectUrl } from '@automattic/jetpack-components';
+import { ExternalLink } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { registerJetpackBlockFromMetadata } from '../../shared/register-jetpack-block';
+import metadata from './block.json';
+import edit from './edit';
+import transforms from './transforms';
 
-registerJetpackBlock( name, settings );
+import './editor.scss';
+
+/**
+ * Supports and extensions
+ */
+import './supports';
+import './extensions/ai-assistant';
+import './extensions/jetpack-contact-form';
+
+registerJetpackBlockFromMetadata( metadata, {
+	edit,
+	save: () => null,
+	// The `description` property in `block.json` doesn't support formatting or markup.
```

Comment:
I wonder if we should stop adding any kind of non-string in there, for all our blocks? That would allow us to close #26792 in the process. :)

Reply:
I originally removed all links since the block description component in the block inserter cannot be hovered, and thus, users could not interact with the links. But then I realized users could click on the links from the Jetpack sidebar and added them back.

But now we have a valid reason to keep it simple! Thanks for the heads-up @jeherve.